src/routes/Intro/Intro.js

```diff
@@ -387,23 +387,23 @@ const Intro = ({ queryParams }) => {
                     {
                         state.form === SIGNUP_FORM ?
                             <Button className={classnames(styles['form-button'], styles['login-form-button'])} onClick={switchFormOnClick}>
-                                <div className={classnames(styles['label'], styles['uppercase'])}>{t('LOG_IN')}</div>
+                                <div className={classnames(styles['label'], styles['lowercase'])}>{t('LOG_IN')}</div>
```

Comment:
here the only thing you need to do i just remove the class entirely from `Intro.js` and `styles.less` which is corresponding to `Intro.js`
```suggestion
                                <div className={styles['label']}>{t('LOG_IN')}</div>
```

Reply:
Okk I will do that.

Reply:
please also remove unused class from `styles.less` as i mentioned before

Reply:
Apologies, should be good now